core/includes/image.inc

```diff
@@ -577,30 +577,39 @@ function image_is_svg($uri) {
  *   which is the accessible mechanism for alternative text within SVGs.
  *
  * @return string
- *   The SVG image contents with the attributes added.
+ *   The SVG image contents with the attributes added. An empty string in case
+ *   of errors.
```

Comment:
Same as previous comment, we should add an `@since` here, to denote the change.

Reply:
You want an `@since` for a comment extension?